tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compilerOptions": {
+        "incremental": true,
+        "target": "ESNext",
```

Comment:
I dont think ESNext is compatible with Node 6 ???

Reply:
The active LTS is 14 if you need node 6 or so babel is the way to go

Reply:
I dont think we should racing with node versions, Instead we should look features what each version offers.

With typescript you can code with latest language features but support older versions as well. Typescript allows library use with Deno as well.

Reply:
It's not really racing just using the active LTS as a target. And with new features we could trade compatibility with performance and readability

Deno doesn't support npm modules natively and the api we are using is probably also not available

Reply:
Sorry I still dont get it, I would like to keep next npm versions compatible with previous adm-zip versions if it is not STRICTLY needed for some feature, but with typescript, anyone who wants can compile it for node 16 API.

Yes, Deno is not aim in any way.